### DISCUSSION TITLE
Unable to update to raven client 5.4 - Preview features required

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#14576] [Q&A] [answered]
I'm unable to update to the latest 5.4 client. 

The solution does not build. 

It complains that I need to opt-in to preview features. It does not tell me how to do that or why I need to?

**Comment by ayende** [comment, marked as answer]
Sorry about that. We didn't notice that this is a transitive requirement. 
This will shortly be fixed and a new stable will be out.

**Comment by bobskigit** [comment]
No worries. Microsoft was not good at communicating it

**Reply by ayende** [reply]
A new build is out
